pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/MhloLegalizeUtils.cpp

```diff
@@ -171,7 +171,7 @@ std::vector<Value> getDimSizesOfTensor(
   dimSizes.reserve(rank);
   auto loc = op->getLoc();
   for (auto d : dims) {
-    auto d_size = currentKnowledge.sizes[d];
+    auto dSize = currentKnowledge.sizes[d];
```

Comment:
ditto

Reply:
It has been updated. Please refresh the page.

![image](https://user-images.githubusercontent.com/8392447/176848505-dc8ff4af-4aac-4b9a-ad37-4302dfa4246d.png)